src/debbit.py

```diff
@@ -424,13 +424,24 @@ def __init__(self, name, web_automation, config_entry):
     LOGGER.info('\__,_/\___/_.___/_.___/_/\__/  ' + VERSION)
     LOGGER.info('')
 
-    try:
-        with open(absolute_path('config.txt'), 'r', encoding='utf-8') as config_f:
-            CONFIG = yaml.safe_load(config_f.read())
-    except FileNotFoundError:
-        LOGGER.error('config.txt not found. Please copy and rename sample_config.txt to config.txt. Then, put your credentials and debit card info in config.txt.')
+    files = ['config.yml', 'config.txt']
+    to_open = ''
+
+    for file in files:
+        if (os.path.exists(file)):
```

Comment:
Only small change needed here is to use absolute_path. This is due to Pyinstaller's binary setting the current working directly very strangely on macOS.

`if (os.path.exists(absolute_path(file))):`

Reply:
done - let me know what you think! very much a python newbie, for what it's worth.